src/test/fuzz/fuzz.cpp

```diff
@@ -94,10 +111,28 @@ int main(int argc, char** argv)
     }
 #else
     std::vector<uint8_t> buffer;
-    if (!read_stdin(buffer)) {
+    if (argc <= 1) {
+        if (!read_stdin(buffer)) {
+            return 0;
+        }
+        test_one_input(buffer);
         return 0;
     }
-    test_one_input(buffer);
+    for (int i = 1; i < argc; ++i) {
+        fs::path seed_path(*(argv + i));
+        if (fs::is_directory(seed_path)) {
+            for (fs::directory_iterator it(seed_path); it != fs::directory_iterator(); ++it) {
+                if (!fs::is_regular_file(it->path())) continue;
+                Assert(read_file(it->path(), buffer));
+                test_one_input(buffer);
+                buffer.clear();
+            }
```

Comment:
Is it worth reporting the number of tests run when all the inputs succeed? It's a bit unclear that anything happened otherwise.

Reply:
Added a counter